mui/mui.py

```diff
@@ -149,10 +150,37 @@ def solve(bv: BinaryView):
         )
 
         if dialog.exec() == QDialog.Accepted:
+            if notif.mui_grpc_server_process == None:
+                notif.mui_grpc_server_process = subprocess.Popen(
+                    [Path(__file__).resolve().parent.parent / "muicore_server"]
+                )
             if bv.session_data.mui_client_stub == None:
-                subprocess.Popen([Path(__file__).resolve().parent.parent / "muicore_server"])
-                time.sleep(3)
-                client_stub = ManticoreUIStub(grpc.insecure_channel("localhost:50010"))
+                client_stub = ManticoreUIStub(
+                    grpc.insecure_channel(
+                        "localhost:50010",
+                        options=[
+                            (
+                                "grpc.service_config",
+                                json.dumps(
+                                    {
+                                        "methodConfig": [
+                                            {
+                                                "name": [{"service": "muicore.ManticoreUI"}],
+                                                "retryPolicy": {
+                                                    "maxAttempts": 5,
+                                                    "initialBackoff": "1s",
+                                                    "maxBackoff": "10s",
+                                                    "backoffMultiplier": 2,
+                                                    "retryableStatusCodes": ["UNAVAILABLE"],
+                                                },
```

Comment:
This is repeated below. We should make this into a constant and not repeat ourselves